lab_3_keywords_textrank/main.py

```diff
@@ -81,7 +91,9 @@ def preprocess_text(self, text: str) -> tuple[str, ...]:
             tuple[str, ...]
                 clean lowercase tokens with no stop-words
         """
-        pass
+        tokenized = self._clean_and_tokenize(text)
+        cleaned_text = self._remove_stop_words(tokenized)
+        return cleaned_text
```

Comment:
you can return `self._remove_stop_words(tokenized)` without creating the variable

Reply:
done